### ISSUE TITLE
Do we support the `Deferred` syntax in Puppet 6

### ISSUE THREAD

**Issue by glennsarti** [lingua-pupuli/puppet-editor-services#95]
Does PES support intellisense on the Deferred keyword;

https://puppet.com/docs/puppet/6.0/write_a_puppet_function_to_store_secrets.html

**Comment by glennsarti** [comment]
Appears we do not.

```
E, [2019-01-23T20:15:41.338863 #11780] ERROR -- : (textDocument/hover) Unable to generate Hover information for object of type Puppet::Pops::Model::QualifiedReference
D, [2019-01-23T20:15:41.339341 #11780] DEBUG -- : --- OUTBOUND
```

**Comment by glennsarti** [comment]
Best example of text I could find

https://github.com/puppetlabs/puppet/pull/6951/commits/cd5c15cffcba90a0cf76eb61b3fec2016bf498a7

**Comment by glennsarti** [comment]
And same with Sensitive

**Comment by glennsarti** [comment]
What about;

```
file { 'title':
  ensure => Sensitive(<crtl-space here>)
}
```

**Comment by jpogran** [comment]
Pending 1.0 work. Needs the infrastructure from 1.0 before we can start to address this
